### ISSUE TITLE
A JavaScript error occurred in the main process

### ISSUE THREAD

**Issue by Oleg-Chashko** [AdguardTeam/AdGuardForSafari#400]
AdGuard for Safari version: v.1.8.5 Beta
Safari v.14.0.1 (16610.2.11.51.7)
Operating System: v.11.0.1 Beta (20B5022a)
 **Screenshots:**
<img width="299" alt="Bildschirmfoto 2020-11-10 um 18 13 26" src="https://user-images.githubusercontent.com/62497891/98708779-d7034080-2381-11eb-85fb-bb46cbe28319.png">
[Bildschirmaufnahme 2020-11-10 um 18.14.54.zip](https://github.com/AdguardTeam/AdGuardForSafari/files/5519068/Bildschirmaufnahme.2020-11-10.um.18.14.54.zip)
[AdGuardForSafariExtension_2020-11-10-180319_MacBook-Pro.crash.zip](https://github.com/AdguardTeam/AdGuardForSafari/files/5519086/AdGuardForSafariExtension_2020-11-10-180319_MacBook-Pro.crash.zip)

**Comment by ameshkov** [comment]
Steps to reproduce?

**Comment by Oleg-Chashko** [comment]
> Steps to reproduce?
I will try to reproduce it, but I do not promise that I will succeed.

There is also such a bug.
**A JavaScript error occurred in the main process. #398**

**Comment by Oleg-Chashko** [comment]
@ameshkov 
If at startup immediately exit.
<img width="302" alt="Bildschirmfoto 2020-11-10 um 18 51 22" src="https://user-images.githubusercontent.com/62497891/98712022-0d42bf00-2386-11eb-98f3-bfe61edab72a.png">

**Comment by Oleg-Chashko** [comment]
@ameshkov
Closed the settings window.
<img width="296" alt="Bildschirmfoto 2020-11-10 um 19 32 40" src="https://user-images.githubusercontent.com/62497891/98716638-336b5d80-238c-11eb-8b83-71709e4fc83f.png">

**Comment by Oleg-Chashko** [comment]
> Steps to reproduce?

Closed the settings window.

**Comment by Oleg-Chashko** [comment]
**Steps to reproduce?**
Closed the settings window. After that, the settings window began to open with a bug.
<img width="258" alt="Bildschirmfoto 2020-11-10 um 19 56 10" src="https://user-images.githubusercontent.com/62497891/98718629-f5bc0400-238e-11eb-985d-dee9501d093f.png">

**Comment by Oleg-Chashko** [comment]
@ameshkov 
Maybe this topic will be useful to you.
Crash and Bugs in macOS Big Sur 11 beta * #379

**Comment by ameshkov** [comment]
> Closed the settings window. After that, the settings window began to open with a bug.

We tried everything, but we simply cannot reproduce this one.

> If at startup immediately exit.

This one will be fixed in the next update.

**Comment by Oleg-Chashko** [comment]
@ameshkov
_Closed the settings window. After that, the settings window began to open with a bug.
We tried everything, but we simply cannot reproduce this one._

I managed to achieve this result twice. 
Perhaps I did not take into account the sequence of actions. 
And you didn't succeed. The load on the computer is different, and so on.

You fix the application, and I will test it and let you know if there is a problem.
If I have time, I will look for problems in the application and if I find I will write.
